src/components/UploadForms/UploadTab.js

```diff
@@ -9,28 +9,48 @@ import UploadPackageInfoForm from './UploadPackageInfoForm';
 import ReviewUpload from './ReviewUpload';
 
 
-const uploader = new FineUploaderTraditional({
-    options: {
-        debug: true,
-        autoUpload: true,
-        maxConnections: 3,
-        chunking: {
-            enabled: true
-        },
-        request: {
-            endpoint: 'http://localhost:3030/upload'
-        },
-        retry: {
-            enableAuto: false
-        }
-    }
-});
+
 
 class UploadTab extends Component {
 
+	constructor(props) {
+		super();
+		this.uploader = new FineUploaderTraditional({

```

Comment:
I was thinking about abstracting this into a new "helper" to grab an instance like I did with the API

Reply:
not a required change just to be clear but if we need to instantiate multiple uploaders it would be nice to have them abstracted

Reply:
I agree...I'll capture it as a technical debt task.